### PR TITLE
Move mod/home to src/Module/Home

### PR DESCRIPTION
**Pull request by nupplaphil**
Part of #4090 
Fixed missing route `/logout` because of #7070

### CODE REVIEW

File: src/Module/Home.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+use Friendica\Core\Hook;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+
+/**
+ * Home module - Landing page of the current node
+ */
+class Home extends BaseModule
+{
+	public static function init()
```
Comment (MrPetovan) [inline src/Module/Home.php]: No `post()` method, you can merge `init()` and `content()`.

Comment (nupplaphil) [reply src/Module/Home.php]: But what about the hook call in the init? At least startpage is using this hook.

Comment (MrPetovan) [reply src/Module/Home.php]: Keep the hook, ditch `init()`.

**Comment by AlfredSK** [comment]
home.html doesn't work anymore! It is not loaded.

**Comment by nupplaphil** [comment]
Ahhh, I see why, I'll fix it

**Pull request merged**